### DISCUSSION TITLE
Where is the alternate speed limit located

### DISCUSSION THREAD

**Discussion by kingp0dd** [Bartuzen/qBitController#126] [Q&A] [answered]
I found in the change log that it has moved but i can't seem to find it

**Comment by Bartuzen** [comment, marked as answer]
On the main screen, right above torrents, there's a text that displays current downloading and uploading speeds. Click on the text and open the speed limits dialog. You can toggle alternative speed limits there.

**Comment by kingp0dd** [comment]
Wow thanks mate, been finding it in the menus for 10mins
